### PR TITLE
clean up mkdoc!

### PR DESCRIPTION
**Pull request by butlerx**

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -5,67 +5,67 @@ strict: True
...
-    - TODO-2017: plans/TODO-2017.md
     - Project Ideas: plans/project-ideas.md
+    - TODO-2017: plans/TODO-2017.md
   - Services:
     - Backuppc: services/backuppc.md
-    - Bitlbee: services/bitlbee.md
     - Bind9: services/bind9.md
+    - Bitlbee: services/bitlbee.md
     - Dovecot: services/dovecot.md
-    - hey: services/hey.md
```
Comment (VoyTechnology) [inline mkdocs.yml:15]: The idea was to have them arranged somewhat alphabetically

File: mkdocs.yml [hunk 2]
```diff
@@ -5,67 +5,67 @@ strict: True
...
+    - Before New Server Purchase: procedures/stuff_that_needs_doing_before_we_can_buy_new_servers.md
     - Committee Changeover: procedures/committeechangeover.md
     - DRAC Access: procedures/dracaccess.md
-    - Gratuitous ARP: procedures/gratuitousarp.md
     - GPGKeys: procedures/gpgkeys.md
+    - Gratuitous ARP: procedures/gratuitousarp.md
     - IRC Operations: procedures/irc_operator.md
     - LDAP Setup: procedures/ldapsetup.md
-    - ldapchshchfn: procedures/ldapchshchfn.md
+    - LDAP Testing: procedures/testing.md
```
Comment (VoyTechnology) [inline mkdocs.yml]: Can the file be renamed to something more descriptive than "testing"?

File: mkdocs.yml (hunk 1 above)
Comment (butlerx) [reply mkdocs.yml:15]: this is alphabetic i just ran sort on them

File: mkdocs.yml (hunk 2 above)
Comment (butlerx) [reply mkdocs.yml]: i didnt name the file

Comment (zergless) [reply mkdocs.yml]: ldaptesting?

**Review by VoyTechnology — APPROVED** [review]

**Pull request merged**